scripts/prepare-release.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# for some reason you can't just put a heredoc in a variable...
```

Comment:
You could make this into a shell function, like
```
usage () {
  echo "$(basename $0) VERSION [PACKAGE...]
  Prepares to release PACKAGEs at VERSION. If no PACKAGEs are provided,
  prepares to release the default packages."
}
```

and then just say `usage` later.